test/Vagrantfile

```diff
@@ -10,7 +10,7 @@ $SERVER_BOX= "cilium/ubuntu"
 $SERVER_VERSION="37"
 
 # RAM and CPU settings
-$MEMORY = (ENV['MEMORY'] || "2048").to_i
+$MEMORY = (ENV['MEMORY'] || "4096").to_i
```

Comment:
I would suggest not change the default value to 4GB here. If you need 4GB in your scenario, you can use the ENV variable (I'm using direnv for this things). If you want to have 4GB in Jenkins, Jenkinsfile is a better place. 
Example: https://github.com/cilium/cilium/blob/master/ginkgo-kubernetes-all.Jenkinsfile#L16

Reply:
There are three of the ginkgo Jenkinsfiles. Can you tell me which of those will do "make" to build Cilium?

Reply:
Does this also mean that the dev VM needs more memory too?